lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -76,11 +76,9 @@ def __init__(
         VisibilityTransformer,
         requested_field_name='visible_to_staff_only',
     ),
-    SupportedFieldType(
-        BlockCompletionTransformer.COMPLETION,
-        BlockCompletionTransformer,
-        'completion'
-    )
+    SupportedFieldType(BlockCompletionTransformer.COMPLETION),
+    SupportedFieldType(BlockCompletionTransformer.COMPLETE),
+    SupportedFieldType(BlockCompletionTransformer.RESUME_BLOCK),
```

Comment:
I'm not personally sure exactly what these changes do, so I'd like someone else to comment on them, but they seem fine to me.